lib/orderbook/OrderBook.ts

```diff
@@ -62,9 +62,9 @@ class OrderBook extends EventEmitter {
   /** Max time for addOwnOrder iterations (due to swaps failures retries). */
   private static MAX_ADD_OWN_ORDER_ITERATIONS_TIME = 10000; // 10 sec
```

Comment:
Not related to this PR but since when are we using this capitalizing standard? And why it is not `readonly`?

Reply:
I couldn't find a standard convention for TypeScript, but I know that the UPPER_SNAKE_CASE is common across programming languages for final constants like this one.

Still, `readonly` wpuld make sense. I checked and right now the compiler will let me reassign these static fields, which clearly we don't want. Also the naming style isn't totally consistent. I'll open an issue.